### ISSUE TITLE
delete cloud object doesn't work

### ISSUE THREAD

**Issue by gilv** [pywren/pywren-ibm-cloud#342]
method `delete_cloudobject` from `storage/utils` throw an exception
```
Traceback (most recent call last):
  File "<string>", line 16, in <module>
AttributeError: 'InternalStorage' object has no attribute 'storage_handler'
```

**Comment by gilv** [comment]
@omerb01 @JosepSampe before we fixing it - what commit or pull request caused it?

**Comment by gilv** [comment]
I guess this one https://github.com/pywren/pywren-ibm-cloud/pull/330

**Comment by omerb01** [comment]
@gilv if you have tried to run EMBL workload with current master branch, there should be couple of changes to apply so the workload will use the new external storage class

**Comment by gilv** [comment]
@omerb01 it's code in PyWren repository that doesn't work

**Comment by omerb01** [comment]
@gilv please share a simple example to reproduce your issue, I'll debug it and will submit a fix

**Comment by omerb01** [comment]
@gilv fix in #344

**Comment by gilv** [comment]
https://github.com/pywren/pywren-ibm-cloud/commit/214116ee24156d5eafd587c9b722949c8066a39a
